searx/settings_defaults.py

```diff
@@ -201,6 +201,7 @@ def apply_schema(settings, schema, path_list):
         'infinite_scroll': SettingsValue(bool, False),
         'cache_url': SettingsValue(str, 'https://web.archive.org/web/'),
         'search_on_category_select': SettingsValue(bool, True),
+        'hotkeys': SettingsValue(str, 'default'),
```

Comment:
You can specify the list of values instead of `str`:
```python
'hotkeys': SettingsValue(('default', 'vim'), 'default'),
```

Reply:
Indeed, that's better. Done!